fmn/delivery/backends/sse.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import absolute_import, unicode_literals
+
+from gettext import gettext as _
+import logging
+
+import pika
+import fedmsg.meta
+from pika.adapters import twisted_connection
+from twisted.internet import reactor, protocol, defer
+
+from .base import BaseBackend
+
+
+_log = logging.getLogger(__name__)
+
+
+class SSEBackend(BaseBackend):
+    """
+    This backend is responsible for the creation of RabbitMQ exchanges and
+    queues and the re-publication of messages for consumption by the Server-
+    Sent Events server.
+
+    Messages are pulled off the "backend" queue and passed to this class, which
+    formats them appropriately and pushes them back onto the appropriate
+    exchange.
+    """
+    __context_name__ = "sse"
+
+    def __init__(self, config):
+        super(SSEBackend, self).__init__(config)
+
+        self.expire_ms = int(self.config.get('fmn.pika.msg_expiration', 3600000))
+        self.amqp_host = self.config.get('fmn.pika.host', 'localhost')
+        self.amqp_port = int(self.config.get('fmn.pika.port', 5672))
```

Comment:
I recommend considering making a central config processor that does the defaults and type conversions at some point (not in this already huge PR) - it saves on mistakes and copying code (if a setting is used in more than one place).